lib/refinement/analyzer.rb

```diff
@@ -93,9 +94,11 @@ def filtered_scheme(scheme_path:, change_level: :full_transitive, filter_when_sc
           suite_name = buildable_reference.attributes['BlueprintName']
           if (change_reason = changes_by_suite_name[suite_name])
             puts "#{suite_name} changed because #{change_reason}" if log_changes
+            each_target&.call(type: :changed, target_name: suite_name, change_reason: change_reason)
```

Comment:
Does the passed in Proc have to have this exact call signature or else it borks?

Reply:
I _think_ so yes?

Reply:
yes it does at least in a quick test I did, thats probably fine.